package.json

```diff
@@ -247,6 +252,12 @@
         "category": "ONE"
       }
     ],
+    "keybindings": [
+      {
+          "command": "one.explorer.runSingleSelectedCfg",
+          "key": "ctrl+f5"
```

Comment:
```suggestion
          "key": "ctrl+f5",
          "when" : "OneExplorerView.active && one.explorer:hasSingleSelectedCfg"
```

Reply:
But IMHO doing this way there is no chance for a user to get knowledge what actions to do for run configuration using shortcut (see https://github.com/Samsung/ONE-vscode/pull/1489#discussion_r1094316314).

Reply:
@dayo09 
IMHO even when there is just a single but not selected .cfg file, some users may still want to run it anyway using shortcut, because there is just a single .cfg file. IMHO. May be just 'OneExplorerView.active" would be sufficient to provide a user some hints how to run .cfg file ?

Reply:
Oh.... Sure :-D Maybe we can change this to one.explorer:hasSelectedCfg or so. 

IMO, anyways enabling this only for cfg nodes would make sense. 
Then maybe we can guide our users about the number of the selected cfgs, like the way the current pr does.

Reply:
ok

Reply:
> IMHO even when there is just a single but not selected .cfg file, some users may still want to run it anyway using shortcut, because there is just a single .cfg file. IMHO. May be just 'OneExplorerView.active" would be sufficient to provide a user some hints how to run .cfg file ?

Oh I just checked this comment.

> IMHO. May be just 'OneExplorerView.active" would be sufficient to provide a user some hints how to run .cfg file ?

I recommend putting `(ctrl+f7)` next to `run cfg` icon (play icon). That way, user will directly notice that the shortcut exists. 

>  IMHO even when there is just a single but not selected .cfg file, some users may still want to run it anyway using shortcut, because there is just a single .cfg file. 

You mean the case of there is only one cfg file in the workspace and user hit the shortcut? I think the case could be supported but I think we can discuss about this in another pr or issue.

Reply:
Ok. I'm working on it.

Reply:
@dayo09 

> > IMHO. May be just 'OneExplorerView.active" would be sufficient to provide a user some hints how to run .cfg file ?
> 
> I recommend putting `(ctrl+f7)` next to `run cfg` icon (play icon). That way, user will directly notice that the shortcut exists. 

Apart from this issue, 
in ONE explorer, it uses ▶️ icon to run cfg. If the `run tvn` feature is introduced afterwards, the meaning of `run` may be a little confusing. If there is a good icon to represent compilation, it would be nice to use this icon for `run cfg` command. :)

Reply:
@jyoungyun How do you expect `run tvn` would be like? Sorry that I don't have any idea about that. I thought that it would be about cfg, not `run cfg`. I would appreciate your explanation, maybe in another issue.